pkgs/games/azimuth/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv, fetchFromGitHub, SDL }:
+
+stdenv.mkDerivation rec {
+  name = "azimuth";
+  version = "1.0.2";
+
+  src = fetchFromGitHub {
+    owner  = "mdsteele";
+    repo   = "azimuth";
+    rev    = "v${version}";
+    sha256 = "0yh52i3vfmj5zd7fs1r2xpjy2mknycr5xz6kyixj2qncb25xsm7z";
+  };
+
+  preConfigure = ''
+    substituteInPlace data/azimuth.desktop \
+      --replace Exec=azimuth "Exec=$out/bin/azimuth" \
+      --replace "Version=%AZ_VERSION_NUMBER" "Version=${version}"
+  '';
+
+  makeFlags = [
+    "BUILDTYPE=release"
+  ];
+
+  buildInputs = [ SDL ];
+
+  enableParallelBuilding = true;
+
+  # the game doesn't have an installation procedure
+  installPhase = ''
+    mkdir -p $out/bin
+    cp out/release/host/bin/azimuth $out/bin/azimuth
+    cp out/release/host/bin/editor $out/bin/azimuth-editor
+    cp out/release/host/bin/muse $out/bin/azimuth-muse
+    cp out/release/host/bin/zfxr $out/bin/azimuth-zfxr
```

Comment:
You can simplify this by doing:

```
cp out/release/host/bin/* $out/bin/
```

Reply:
@matthiasbeyer if you look closely you'll notice the binaries are being renamed.